criu/mount.c

```diff
@@ -1228,7 +1228,7 @@ int __open_mountpoint(struct mount_info *pm)
 		return -1;
 	}
 
-	if (check_mountpoint_fd(pm, mnt_fd)) {
+	if (__check_mountpoint_fd(pm, mnt_fd, false)) {
```

Comment:
There is no point in doing this change, unless you want to remove check_mountpoint_fd completely...

Reply:
It seems you broke tests with this one. Please just drop hunk in that place.